walkthroughs/week-2-web-development/portfolio-walkthrough.md

```diff
@@ -53,31 +53,44 @@ You'll learn more about these files in the following steps.
 
 ## Java 8
 
-Before you continue, set your default Java version to Java 8 by running this
-command:
+Before you continue, set your default Java version to Java 8.
```

Comment:
I'm assuming these are some updates from the googleinterns/step repo? Please also update your master branch so that these files don't show up in the PR.